fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java

```diff
@@ -97,6 +103,9 @@
     private final SavingsAccountTransactionRepository savingsAccountTransactionRepository;
     private final AccountTransfersReadPlatformService accountTransfersReadPlatformService;
     private final ChargeRepositoryWrapper chargeRepositoryWrapper;
+    private final JdbcTemplate jdbcTemplate;
+    private final DataSource dataSource;
+    private static final Logger LOG = LoggerFactory.getLogger(AccountingProcessorHelper.class);
```

Comment:
Hi @BLasan.

Are these 3 attributes used anywhere in the code? Right now I don't see any usage. If they aren't, do you think it's worth removing them?
Thanks!